src/components/spreadsheet/custom-spreadsheet/custom-spreadsheet-dialog.tsx

```diff
@@ -137,6 +137,7 @@ export default function CustomSpreadsheetConfigDialog({
                                     selectedModel.customColumns.map((col) => {
                                         return {
                                             ...col,
+                                            uuid: crypto.randomUUID(),
```

Comment:
Actually in other repos we rather use import { v4 as uuid4 } from 'uuid'; to generate uuids. Can you also fix it in other places it appears here ? We introduced them recently